src/features/events/components/EventParticipantsTab/AddPersonButton.tsx

```diff
@@ -0,0 +1,138 @@
+import PersonAddIcon from '@mui/icons-material/PersonAdd';
+import { Box, IconButton, Popover } from '@mui/material';
+import { EmojiPeople, People } from '@mui/icons-material';
+import { useRef, useState } from 'react';
+
+import EventDataModel from 'features/events/models/EventDataModel';
+import messageIds from '../../l10n/messageIds';
+import ZUIFutures from 'zui/ZUIFutures';
+import { MUIOnlyPersonSelect as ZUIPersonSelect } from 'zui/ZUIPersonSelect';
+import { Msg, useMessages } from 'core/i18n';
+
+interface AddPersonButtonProps {
+  model: EventDataModel;
+}
+
+const AddPersonButton = ({ model }: AddPersonButtonProps) => {
+  const [anchorEl, setAnchorEl] = useState<Element | null>(null);
+
+  const messages = useMessages(messageIds);
+
+  const handleSelectedPerson = (personId: number) => {
+    model.addParticipant(personId);
+  };
+
+  const selectInputRef = useRef<HTMLInputElement>();
```

Comment:
What's the purpose of this? It doesn't seem to me like it's used anywhere. Is it necessary?

Reply:
It threw this null inputRef for useAutocomplete error that I had before. I checked the view footer, and it also uses selectInputRef like that, so I also copied a same logic from there. It resovled null inputRef error.

Reply:
Are you saying that you introduced it while you were using `useAutocomplete` and it's no longer relevant, or that you introduced it then but you still need it? 

Is it not possible to use `ZUIPersonSelect` without a ref? It should be, so if you are experiencing that it's not possible, that must be a bug with the component. Maybe this is a fine workaround, but I don't understand if it is a workaround for an actual bug, or just a mistake.

Reply:
Okay! I want to ask you about these errors tomorrow then! I just want to assure having these errors are still okay.